.appveyor.yml

```diff
@@ -1,118 +1,119 @@
-    SDL_AUDIODRIVER: "disk"
- 
-  matrix:
-    # - PYTHON: "pypy.exe"
-    #   PYTHON_VERSION: "2.7.13"
-    #   PYTHON_ARCH: "32"
-    #   PYPY_VERSION: "pypy2-v5.10.0-win32"
-    #   PYPY_DOWNLOAD: "powershell buildconfig\\ci\\appveyor\\download_pypy.ps1"
-
-    # - PYTHON: "pypy3.exe"
-    #   PYTHON_VERSION: "2.7.13"
-    #   PYTHON_ARCH: "32"
-    #   PYPY_VERSION: "pypy3-v5.10.1-win32"
-    #   PYPY_DOWNLOAD: "powershell buildconfig\\ci\\appveyor\\download_pypy.ps1"
-
-    # - PYTHON: "C:\\Python27\\python"
-    #   PYTHON_VERSION: "2.7.13"
-    #   PYTHON_ARCH: "32"
-    #   USE_SDL2: "-sdl2"
-
-    # - PYTHON: "C:\\Python37\\python"
-    #   PYTHON_VERSION: "3.7.0"
-    #   PYTHON_ARCH: "32"
-    #   USE_SDL2: "-sdl2"
-
-    # - PYTHON: "C:\\Python27-x64\\python"
-    #   PYTHON_VERSION: "2.7.13"
-    #   PYTHON_ARCH: "64"
-    #   USE_SDL2: "-sdl2"
-
-    # - PYTHON: "C:\\Python37-x64\\python"
-    #   PYTHON_VERSION: "3.7.0"
-    #   PYTHON_ARCH: "64"
-    #   USE_SDL2: "-sdl2"
-
-    - PYTHON: "C:\\Python27\\python"
-      PYTHON_VERSION: "2.7.13"
-      PYTHON_ARCH: "32"
-
-    - PYTHON: "C:\\Python34\\python"
-      PYTHON_VERSION: "3.4.4"
-      PYTHON_ARCH: "32"
-
-    - PYTHON: "C:\\Python35\\python"
-      PYTHON_VERSION: "3.5.2"
-      PYTHON_ARCH: "32"
-
-    - PYTHON: "C:\\Python36\\python"
-      PYTHON_VERSION: "3.6.0"
-      PYTHON_ARCH: "32"
-
-    - PYTHON: "C:\\Python37\\python"
-      PYTHON_VERSION: "3.7.0"
-      PYTHON_ARCH: "32"
-
-    - PYTHON: "C:\\Python27-x64\\python"
-      PYTHON_VERSION: "2.7.13"
-      PYTHON_ARCH: "64"
-
-    - PYTHON: "C:\\Python34-x64\\python"
-      PYTHON_VERSION: "3.4.4"
-      PYTHON_ARCH: "64"
-
-    - PYTHON: "C:\\Python35-x64\\python"
-      PYTHON_VERSION: "3.5.2"
-      PYTHON_ARCH: "64"
-
-    - PYTHON: "C:\\Python36-x64\\python"
-      PYTHON_VERSION: "3.6.0"
-      PYTHON_ARCH: "64"
-
-    - PYTHON: "C:\\Python37-x64\\python"
-      PYTHON_VERSION: "3.7.0"
-      PYTHON_ARCH: "64"
-
-init:
-  - "ECHO %PYTHON% %PYTHON_VERSION% %PYTHON_ARCH%"
-
-install:
-  - "powershell buildconfig\\ci\\appveyor\\install.ps1"
-  - "%PYPY_DOWNLOAD%"
-  - "set HOME=%APPVEYOR_BUILD_FOLDER%"
-  - "set PATH=%PATH%;%APPVEYOR_BUILD_FOLDER%\\%PYPY_VERSION%"
-  - "%PYTHON% -m ensurepip"
-  - "%PYTHON% -m pip install -U wheel pip requests"
-  - "%WITH_COMPILER% %PYTHON% -m buildconfig %USE_SDL2%"
-  - "%WITH_COMPILER% %PYTHON% setup.py build"
-  - "%WITH_COMPILER% %PYTHON% setup.py -setuptools %DISTRIBUTIONS%"
-  - ps: "ls dist"
-
-  # Install the wheel to test it
-  - "%PYTHON% -m pip install --ignore-installed --pre --no-index --find-links dist/ pygame"
-
-# Appveyor's build step is specific to .NET projects, so we build in the
-# install step instead.
-build: off
-
-test_script:
-  - "mkdir empty_folder"
-  - "cd empty_folder"
-  - "%PYTHON% -m pygame.tests --exclude opengl"
-
-  # Move back to the project folder
-  - "cd .."
-
-artifacts:
-  - path: 'dist\*'
+environment:
```

Comment:
Hrmm. What changed in this file? It seems all the lines were changed for some reason, but I can't see what the changes were.

Reply:
the file in **master** has `\r`n` at end of each line, maybe that?

Reply:
I only added `PYTHONIOENCODING: "UTF-8"`. AppVeyor reroutes stdout, and it caused some issue. But I'm not sure if it's needed anymore.

Reply:
Ah, ok. I guess the editor (on windows) changed all the line endings.